README.md

```diff
@@ -102,6 +102,7 @@ a double dash to prevent input files being used as option arguments:
                                 sequences.
     --config-file <file>        Read `minify()` options from JSON file.
     -d, --define <expr>[=value] Global definitions.
+    --ecma <version>            Specifiy supported level of ECMAScript.
```

Comment:
> yet it follows Principle of Least Surprise

I think it will do what most people expect. They have to take an explicit action to produce non-ES5 compliant code for ES5 input.

Alternatively, we can perform a pass in the compressor and/or parser to detect the presence of ES6+ features and set the compress/output `ecma` option(s) accordingly. But that takes effort.